client/src/app/containers/Suggestions/ChoosingCard.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
```

Comment:
:cactus: These should prolly just use the named imports from `'@material-ui/core`